p5js/LSTM_Interactive/index.html

```diff
@@ -13,7 +13,8 @@
 
   <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.6.0/p5.min.js"></script>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.6.0/addons/p5.dom.min.js"></script>
-  <script src="https://unpkg.com/ml5@0.0.3/dist/ml5.js"></script>
+  <!-- <script src="https://unpkg.com/ml5@0.0.3/dist/ml5.js"></script> -->
+  <script src="../../temp_lib/ml5.min.js"></script>
```

Comment:
This is b/c we haven't done the release yet?

Reply:
yep, just temporary